builders/interoperability/xcm/remote-execution/remote-evm-calls.md

```diff
@@ -22,29 +22,26 @@ Moonbeam的EVM仅能通过[Ethereum Pallet](https://github.com/paritytech/fronti
-- **交换信息** — 与XCM Transactor extrinsic的XCM远程执行部分的额外权重和费用信息有关。这部分为必要的，因主权账户将支付XCM交易费用。因此，XCM Transactor就计算费用数值，并向XCM Transactor extrinsic的发送方收取相应[XC-20 Token](/builders/interoperability/xcm/xc20/overview/){target=\_blank}的估计数量以偿还主权账户
-
 ## 通过XCM执行常规和远程EVM调用的差异 {: #differences-regular-remote-evm}
 
 正如[概览](#introduction)中所解释的，常规和远程EVM调用到达EVM的路径是完全不同的，其主要原因是交易的调度程序。
 
 一个常规的EVM调用有一个明显的发送者使用其私钥签署以太坊交易。该签名，也就是ECDSA类型的签名，可以通过签名消息和签名算法生成的`r-s`值进行验证。而以太坊签名使用一个额外的变量，称为`v`，为恢复标识符。
 
+With remote EVM calls, the signer signs an XCM transaction in another chain. Moonbeam receives that XCM message, which follows the conventional remote execution via XCM form:
+
 通过远程EVM调用，签名者在另一条链中签署XCM交易。Moonbeam接收到后，必须使用以下指令构建的XCM消息如下：
 
-- [`DescendOrigin`](https://github.com/paritytech/xcm-format#descendorigin){target=\_blank}
-- [`WithdrawAsset`](https://github.com/paritytech/xcm-format#withdrawasset){target=\_blank}
-- [`BuyExecution`](https://github.com/paritytech/xcm-format#buyexecution){target=\_blank}
-- [`Transact`](https://github.com/paritytech/xcm-format#transact){target=\_blank}
+- [`DescendOrigin`](/builders/interoperability/xcm/core-concepts/instructions#descend-origin){target=\_blank} (optional)
+- [`WithdrawAsset`](/builders/interoperability/xcm/core-concepts/instructions#withdraw-asset){target=\_blank}
+- [`BuyExecution`](/builders/interoperability/xcm/core-concepts/instructions#buy-execution){target=\_blank}
+- [`Transact`](/builders/interoperability/xcm/core-concepts/instructions#transact){target=\_blank}
+
+XCM execution happens through a [Computed Origin account mechanism](/builders/interoperability/xcm/remote-execution/computed-origins){target=\_blank}, which by default uses the source chain's Sovereign account in the destination chain. If `DescendOrigin` is included, Moonbeam will mutate the origin of the XCM call to a keyless account that a user from the source chain can control remotely via XCM. The remote EVM call is dispatched from that keyless account (or a related [proxy](/tokens/manage/proxy-accounts/){target=\_blank}). Therefore, because the transaction is not signed, it does not have the real `v-r-s` values of the signature, but `0x1` instead.
 
 第一条指令`DescendOrigin`将通过[XCM相关定义部分](#general-xcm-definitions)中描述的**Multilocation衍生账户**机制在Moonbeam端转移XCM调用的源头。远程EVM调用是从该无密钥帐户（或相关的[代理](/tokens/manage/proxy-accounts/){target=\_blank}）派遣的。因此，由于交易并没有签署，它没有真正签名的`v-r-s`值，反而会是`0x1`。
```

Comment:
Looks like these sections aren't 1:1. Can you please provide the translation of the content on line 44? @wuzhong-papermoon